docker/crashpad_symbolize/Dockerfile

```diff
@@ -0,0 +1,56 @@
+FROM cobalt-base
+
+RUN apt update -qqy \
+    && apt install -qqy --no-install-recommends jq build-essential zlib1g-dev libzstd-dev \
+    && /opt/clean-after-apt.sh
+
+RUN cd /tmp \
+  && git clone 'https://chromium.googlesource.com/chromium/tools/depot_tools.git'
+
+ENV PATH=${PATH}:/tmp/depot_tools
+
+RUN mkdir breakpad \
+  && cd breakpad \
+  && fetch breakpad \
+  && cd src \
+  && ./configure \
+  && make
+
+ARG GITHUB_TAG
```

Comment:
I'd suggest taking everything below from here and moving it into a python script that gets added to the container, and does all the following bits at the container run time, rather than build time. Again - this can be a later refactor.

Reply:
See my above comment, I think this is better at build time for partners